Update README.md

foovungle opened the pull request:
Connector -> CassandraConnector in the documentation

alexflav23 commented on README.md:
````diff
@@ -130,7 +130,7 @@ performance improvement over query performance. Async iterators needed a lot of
 
 Phantom connectors now require an ```implicit com.websudos.phantom.connectors.KeySpace``` to be defined. Instead of using a plain string, you just have to
-use ```KeySpace.apply``` or simply: ```trait MyConnector extends Connector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
+use ```KeySpace.apply``` or simply: ```trait MyConnector extends CassandraConnector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
````
Thank you for correcting that. I think we may better go with `SimpleCassandraConnector`, as that will only require defining an implicit keyspace into scope and it's already available in the default imports.

foovungle replied:
That's a good point. Disregard my pull request, but please correct the README. Took me a little bit of hunting to find the right class to extend.

alexflav23 replied:
@foovungle Fair game, we are working on a brand new wiki for Phantom which will do a far better job at introducing you to phantom + several tutorials.

alexflav23 commented:
Hi @foovungle , we've moved all documentation to Github Wiki and fixed all broken links in the process. I will now close this PR as it's out of date.

foovungle commented:
Thanks

On Saturday, May 30, 2015, Flavian Alexandru <notifications@github.com>
wrote:

> Hi @foovungle <https://github.com/foovungle> , we've moved all
> documentation to Github Wiki and fixed all broken links in the process. I
> will now close this PR as it's out of date.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/websudos/phantom/pull/245#issuecomment-107081882>.
>
